.github/workflows/buf.yaml

```diff
@@ -16,21 +16,23 @@ jobs:
       - uses: bufbuild/buf-lint-action@v1.1.0
         with:
           input: proto
-  breaking:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: bufbuild/buf-setup-action@v1
-      - uses: bufbuild/buf-breaking-action@v1.1.3
-        with:
-          input: proto
-          against: 'buf.build/connectrpc/conformance'
+  # TODO - Reenable after pushing breaking proto changes to BSR
+  # breaking:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v4
+  #     - uses: bufbuild/buf-setup-action@v1
+  #     - uses: bufbuild/buf-breaking-action@v1.1.3
```

Comment:
I think _this_ is the only line you actually need to comment out, if you wanted to simplify this a little. Fine to leave as is, too.